HHStyle.css

```diff
@@ -208,4 +207,18 @@ aside table .pointsColumn {
 
 .error {
     outline: solid 3px #700b0b;
+}
+
+figcaption {
+    font-weight: bold;
+    color: maroon;
```

Comment:
Don't use named colours - use a hex value. I think I used "maroon" to mean the standards purple-ish red?